src/config.rs

```diff
@@ -34,6 +203,9 @@ pub struct Frame {
     pub lineno: Option<u32>,
     /// source file path
     pub filename: Option<PathBuf>,
+    // XXX is it correct that this is public?
+    /// The global styles that `color_eyre` uses
+    pub styles: Styles,
```

Comment:
I don't think so. I wouldn't expect us to want to put a `Styles` into the `Frame` type at all. Instead I think we should probably use a `StyledFrame(Frame, Styles)` type that provides the `fmt::Display` implementation we use where we bundle the frame and style together right before printing.

Reply:
Ah, yes, that makes much more sense (now, that I've implemented it). When I wrote the first version I didn't get why `Frame` is public and assumed users might want to style it themselves for some reason.